pkgs/development/python-modules/qscintilla-qt4/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, disabledIf
+, isPy3k
+, isPyPy
+, pkgs
+, python
+, pyqt4
+}:
+
+disabledIf (isPy3k || isPyPy)
+  (buildPythonPackage {
+    # TODO: Qt5 support
```

Comment:
```suggestion
```

Reply:
Qt5 support dropped in the latest additional commits!